src/Kopernicus/Components/KSC.cs

```diff
@@ -100,6 +100,10 @@ public class KSC : SerializableMonoBehaviour
         [SerializeField]
         public Color? color;
 
+        // Grass Material
+        [SerializeField]
+        public GrassMaterial Material = new GrassMaterial();
```

Comment:
I've seen other commits to the effect that behavior like this causes exceptions in KSP 1.8's version of Unity (e.g. https://github.com/snjo/Firespitter/pull/219/files#diff-f12b9be8c1e03fbc2ba80828371d75f4).  So maybe this should have ```public GrassMaterial Material;``` up here and then instantiate ```material = new GrassMaterial();``` in ```Awake()```?

Reply:
sure, seems reasonable

Reply:
fixed